python/demo/demo_stokes.py

```diff
@@ -636,23 +641,28 @@ def mixed_direct():
 
 
 # Solve using LinearProblem class
+
```

Comment:
I don't think the extra line breaks help. Association with code is clearer without.

Reply:
They do when you look at the web-page, as it currently looks as:
<img width="783" height="453" alt="image" src="https://github.com/user-attachments/assets/069d8cb7-843d-487f-b35b-cabd63953821" />
while with the linebreaks it looks like
<img width="783" height="707" alt="image" src="https://github.com/user-attachments/assets/61f748c0-0b0a-4e0b-9afc-29afc9c77e47" />